app/frontend/src/cljs/teet/common/common_styles.cljs

```diff
@@ -524,6 +524,7 @@
     :warning [theme-colors/dark-tangerine-11 theme-colors/dark-tangerine-1]
     :error [theme-colors/red theme-colors/red-lightest]
     :info [theme-colors/blue theme-colors/blue-lightest]
+    :simple [theme-colors/white theme-colors/white]
```

Comment:
I think this is unnecessary as you shouldn't use the info-box when you don't want to see the borders?